data/Vishal21121.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Vishal Singh",
+  "displayStatsPublic": true,
```

Comment:
Thank you for submitting this pull request. I noticed that the `avatar` and `displayStatsPublic` properties are no longer used. Would it be possible to remove it from the pull request to keep everything up to date? 
```suggestion
 
```